docs/tutorial/asar-integrity.md

```diff
@@ -53,3 +63,71 @@ flipFuses(
   }
 )
 ```
+
+:::tip Fuses in Electron Forge
+
+With Electron Forge, you can configure your app's fuses with [@electron-forge/plugin-fuses]
+(https://www.electronforge.io/config/plugins/fuses) in your Forge configuration file.
+
+:::
+
+## Providing the header hash
+
+ASAR integrity validates the contents of the ASAR archive against the header hash that you provide
+on package time. The process of providing this packaged hash is different for macOS and Windows.
+
+### Using Electron tooling
+
+Electron Forge and Electron Packager do this setup automatically for you with no additional
+configuration. The minimum required versions for ASAR integrity are:
+
+* `@electron/packager@18.3.1`
+* `@electron/forge@7.4.0`
```

Comment:
Should this be blocked on this version of Forge actually being released? Also, the current unreleased commits on Forge seem to indicate a release today would be 7.3.2 (no feats).

Reply:
Semi-related, I wonder if we should do a minor for something like the Windows ASAR integrity 🤔 Do we know which PR ensured the right version of Packager, I think this one?: https://github.com/electron/forge/pull/3551

Reply:
Yeah, that's the one.

Reply:
I think bumping for 7.4.0 would make sense here for ASAR integrity :)